lib/components/src/addon_panel/__snapshots__/index.stories.storyshot

```diff
@@ -5,29 +5,29 @@ exports[`Storyshots Components|AddonPanel default 1`] = `
   style="height:calc(100vh - 20px)"
 >
   <div
-    class="css-d7g4da"
+    class="css-d7g4da e1lzzkxx0"
   >
     <div
-      class="css-1whcqvw"
+      class="css-1whcqvw e1lzzkxx1"
       role="tablist"
     >
       <button
-        class="css-xcko46"
+        class="css-xcko46 e1lzzkxx2"
         role="tab"
         type="button"
       >
         Test 1
       </button>
       <button
-        class="css-1hf831j"
+        class="css-1hf831j e1lzzkxx2"
         role="tab"
         type="button"
       >
         Test 2
       </button>
     </div>
     <div
-      class="css-18lpvrh"
+      class="css-18lpvrh e1lzzkxx3"
```

Comment:
@Hypnosphi what's up with all these extra CSS classes? Where do they come from? 🤔

Reply:
No idea, something from emotion probably

Reply:
This PR updates `babel-plugin-emotion` among other things, so that may be it